src/main/resources/project/ec_setup.pl

```diff
@@ -86,6 +86,12 @@
     description => "Puppet version",
     category    => "Resource Management"
 );
+my %PuppetAgentSetConfig = (
+    label       => "Puppet - AgentSetConfig",
```

Comment:
The name of the procedure is a little confusing. Lets call it 'UpdateAgentConfiguration'. The pattern <Verb>+<Noun> should be used in general for naming procedures.